dev/docker_image/Dockerfile

```diff
@@ -108,6 +108,9 @@ ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/software/kentUtils_bin_v377/lib
 # Instal Trimmomatic JAR
 RUN wget http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/Trimmomatic-0.39.zip && unzip Trimmomatic-0.39.zip && mv Trimmomatic-0.39/trimmomatic-0.39.jar trimmomatic.jar && chmod +rx trimmomatic.jar && rm -rf Trimmomatic-0.39.zip Trimmomatic-0.39/
 
+# Install pytest for testing environment
+RUN pip3 install --no-cache-dir pytest
```

Comment:
Is there a good reason to add pytest to the production image? It seems like it would be better to leave it out, put it somewhere like a `requirements-dev.txt` or in a separate Dockerfile, then just use that in CircleCI to run the tests.

Reply:
Yes, I first tried to `pip install pytest` on circleci's instance and run `pytest` from there since we have those wrapper on git directory.

But those wrapper `.py`s are assumed to be run in an activated Conda environment or inside a docker container, because they directly call genomic softwares like bowtie2, bedtools and bedClip (USCS tool).

Installing all genomic softwares on a circleci machine was hard. There are limited choices for docker images that circleci can run on. So I couldn't use pipeline's image itself for circleci instance.